src/App/Header/UserMenu/UserMenu.tsx

```diff
@@ -14,20 +14,32 @@ export default function UserMenu() {
 
   const location = useLocation();
 
-  if (!user || user === "loading") {
+  if (!user) {
     const state: SignInRouteState = { from: location.pathname };
     return (
-      <Link
-        to={appRoutes.signin}
-        state={state}
-        className="btn-orange px-3 h-10 rounded-lg text-sm"
-        aria-disabled={user === "loading"}
-      >
-        Sign-up / Login
-      </Link>
+      <>
+        <Link
+          to={appRoutes.signin}
+          state={state}
+          className="btn text-base normal-case max-sm:hidden hover:underline"
+        >
+          Login
+        </Link>
+        <Link
+          to={appRoutes.signin}
+          state={state}
+          className="btn text-base normal-case max-sm:hidden bg-blue-d1 hover:bg-blue text-white text-nowrap px-5 sm:px-6 lg:px-7 py-3 rounded-full"
+        >
+          Sign up
+        </Link>
+      </>
     );
   }
 
+  if (user === "loading") {
```

Comment:
@SovereignAndrey added this loading indicator as it makes it clearer to users that some process is running after they sign in. 
If you prefer the former behavior (while signing in, login/sign up buttons still appear but disabled) will revert this.